src/pages/main-page/OurProjects.tsx

```diff
@@ -8,16 +8,64 @@ import r3 from '../../assets/rockets/r3.png';
 import r4 from '../../assets/rockets/r4.png';
 import r4v2 from '../../assets/rockets/r4v2.png';
 
-const rockets = [
-    'R1',
-    'R2',
-    'R3',
-    'R2S',
-    'R4S',
-    'R4',
-    'R4V2',
-    'R5'
-];
+const rockets = {
+    'R1': {
+        'length': '750 MM',
+        'thrust': '210 N',
+        'weight': '0.8 KG',
+        'velocity': '120 M/S',
+        'image': r1
+    },
+    'R2': {
+        'length': '1900 MM',
+        'thrust': '450 N',
+        'weight': '5.5 KG',
+        'velocity': '160 M/S',
+        'image': r2
+    },
+    'R3': {
```

Comment:
i created an object with each rocket details, that every function iterates over, so it's even easier for someone to add a new rocket - a new button and rocket panel will be automatically created

Reply:
Yes, this is exactly what I wanted to see :)